chart/value.yaml.j2

```diff
@@ -0,0 +1,271 @@
+# Settings template used by ansible
+# =========================================
+name: browsertrix-cloud
+
+# when running in the cloud, set this value to cloud-specific block storage
+# keep empty to use hostPath (eg. on minikube)
+volume_storage_class:
+
+# if set, set the node selector 'nodeType' for deployment pods
+# main_node_type:
+
+# if set, set the node selector 'nodeType' to this crawling pods
+# crawler_node_type:
+
+registration_enabled: "0"
+jwt_token_lifetime_minutes: 1440
+
+# default time to run behaviors on each page (in seconds)
+default_behavior_time_seconds: 300
+
+# default time to wait for page to fully load before running behaviors (in seconds)
+default_page_load_time_seconds: 120
+
+# disk utilization threshold percentage - when used disk space passes
+# this percentage of total, crawls will gracefully stop to prevent the
+# disk from being filled
+disk_utilization_threshold: 90
+
+# max pages per crawl
+# set to non-zero value to enforce global max pages per crawl limit
+# if set, each workflow can have a lower limit, but not higher
+max_pages_per_crawl: 0
+
+# if set to "1", allow inviting same user to same org multiple times
+allow_dupe_invites: "0"
+
+# number of workers for backend api
+backend_workers: 4
+
+# number of seconds before pending invites expire - default is 7 days
+invite_expire_seconds: 604800
+
+# base url for replayweb.page
+rwp_base_url: "https://replayweb.page/"
+
+superuser:
+  # set this to enable a superuser admin
+  email: "{{ browsertrix_superuser_email| default('admin@example.com') }}"
+
+  # optional: if not set, automatically generated
+  # change or remove this
+  password: "{{ browsertrix_superuser_password | default('PASSW0RD!') }}"
+
+# Set name for default organization created with superuser
+default_org: "{{ browsertrix_default_org | default('My Organization') }}"
```

Comment:
We only override these values (and mongodb password, a bit further down) 

I'm thinking we set these as Go Template values, which we can override through ansible passed as a config variable. Something like `helm upgrade --install -f ./chart/values.yaml btrix ./charts --set default_org={{ browsertrix_default_org | default('My Org') }}`

Reply:
Looking at it some more, we already have a jinja template that's generating values. We can leverage that to include these values, and scrap the double template.

Reply:
Is there a way to add overrides in a file some where in some sort of stacked method.

For example in docker-compose.yml you can also do docker-compose.override.yml 

This way ansible can place a list of variable overrides into a file that is normally emtpy.

Settings are more then just passwords, currently this is the config file we are using (passwords removed for this git hub commit)

https://github.com/starlinglab/browsertrix-cloud/blob/starling/ansible/inventory/hosts.yml

Reply:
I don't think it's necessary is more what I was trying to communicate. 
We're already overriding `../group_vars/microk8s/btrix_values.j2` which gets deployed in the same command through `kubectl`. Adding variables that we wish to template from ansible to that file instead of creating a new template should give us the same results? This way it's not necessary to keep both the regular yaml file and it's template up to date.

Reply:
For reference, there's https://github.com/webrecorder/browsertrix-cloud/pull/979 which I'm still testing out.